test/Api.Test/Public/Controllers/CollectionsControllerTests.cs

```diff
@@ -0,0 +1,110 @@
+﻿using Bit.Api.Models.Public.Response;
+using Bit.Api.Public.Controllers;
+using Bit.Core.Context;
+using Bit.Core.Entities;
+using Bit.Core.Enums;
+using Bit.Core.Models.Data;
+using Bit.Core.Repositories;
+using Bit.Test.Common.AutoFixture;
+using Bit.Test.Common.AutoFixture.Attributes;
+using Microsoft.AspNetCore.Mvc;
+using NSubstitute;
+using Xunit;
+
+namespace Bit.Api.Test.Public.Controllers;
+
+[ControllerCustomize(typeof(CollectionsController))]
+[SutProviderCustomize]
+public class CollectionsControllerTests
+{
+    [Theory, BitAutoData]
+    public async Task Get_WithDefaultUserCollection_ReturnsNotFound(
+        Collection collection, SutProvider<CollectionsController> sutProvider)
```

Comment:
Non-blocker: Consider using the [AAA (Arrange, Act, Assert) pattern](https://learn.microsoft.com/en-us/visualstudio/test/unit-test-basics?view=vs-2022#write-your-tests) for unit tests. It helps make tests easier to follow, especially as they get longer.

Reply:
Addressed this in https://github.com/bitwarden/server/pull/6021/commits/e71ce64a0f2889c29c1224b4247ce9cdc579e8e6